src/MaxText/examples/sft_llama3_demo.ipynb

```diff
@@ -251,22 +251,69 @@
         "    print(\"=\"*60)\n",
         "    print(\"EXECUTING ACTUAL TRAINING\")\n",
         "    print(\"=\"*60)\n",
-        "    \n",
-        "    sft_train(config)           \n",
-        "    \n",
+        "\n",
+        "    trainer, mesh = sft_train(config)\n",
+        "\n",
         "    print(\"\\n✅ Training completed successfully!\")\n",
-        "               \n",
+        "\n",
         "else:\n",
         "    print(\"MaxText not available - cannot execute training\")\n"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "##  Decoding\n",
+        "\n",
+        "We could reuse the trainer.model to try a decode command after the SFT."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# Decode by using the trained model\n",
+        "if MAXTEXT_AVAILABLE:\n",
+        "    from MaxText.sft.vllm_decode import decode\n",
+        "\n",
+        "    decode_config_argv = [\n",
+        "        \"\",  \n",
+        "        f\"{MAXTEXT_HOME}/src/MaxText/configs/sft.yml\",   # Decode config\n",
+        "        \"model_name=llama3.1-8b\",\n",
+        "        \"per_device_batch_size=1\",\n",
+        "        \"max_target_length=128\",\n",
+        "        \"max_prefill_predict_length=64\",\n",
+        "        \"weight_dtype=bfloat16\",\n",
+        "        \"dtype=bfloat16\",\n",
+        "        f\"hf_access_token={HF_TOKEN}\",\n",
+        "        \"base_output_directory=/tmp/maxtext_output\",\n",
+        "        \"run_name=sft_llama3_decode\",\n",
+        "        \"tokenizer_path=meta-llama/Llama-3.1-8B-Instruct\",\n",
```

Comment:
For consistency, we should use both the `model_name` and the `tokenizer_path` as the `-Instruct` version